$OpenBSD: patch-requirements_txt,v 1.1 2018/04/25 14:26:26 jasper Exp $

textfsm supports python3 now, so no need to require jtextfsm

Index: requirements.txt
--- requirements.txt.orig
+++ requirements.txt
@@ -1,7 +1,6 @@
 setuptools>=38.4.0
 cffi>=1.11.3
 future
-jtextfsm
 jinja2
 netaddr
 pyYAML
